llama_stack/providers/inline/inference/meta_reference/inference.py

```diff
@@ -375,6 +384,21 @@ def impl():
                         assert len(token_result.logprobs) == 1
 
                         logprobs.append(TokenLogProbs(logprobs_by_token={token_result.text: token_result.logprobs[0]}))
+
+                    if isinstance(delta, InProgressToolCallDelta):
+                        deprecated_delta = ToolCallDelta(
+                            tool_call=delta.tool_call,
+                            parse_status=ToolCallParseStatus.in_progress,
+                        )
+                        yield ChatCompletionResponseStreamChunk(
+                            event=ChatCompletionResponseEvent(
+                                event_type=ChatCompletionResponseEventType.progress,
+                                delta=deprecated_delta,
+                                stop_reason=stop_reason,
+                                logprobs=logprobs if request.logprobs else None,
+                            )
+                        )
```

Comment:
How to handle the deprecated delta that's getting sent in addition to the new delta?